### PR TITLE
update F.kl_div matcher 易用性提升

### PR DESCRIPTION
**Pull request by NKNaN**
### PR Docs

update F.kl_div matcher to support new attribute `log_target` in paddle

### PR APIs

```bash
torch.nn.functional.kl_div
```

### CODE REVIEW

File: paconvert/api_matcher.py [hunk 1]
```diff
@@ -3376,14 +3376,12 @@ def generate_code(self, kwargs):
 
         if "target" in kwargs:
             kwargs["label"] = kwargs.pop("target")
```
Comment (zhwesky2010) [inline paconvert/api_matcher.py:3378]: 如果参数都已经对齐了，建议是不是简化写法，这么写就可以：
```
    def generate_code(self, kwargs):
        process_reduce_and_size_average(kwargs)
        return GenericMatcher.generate_code(self, kwargs)
```
然后再配置相应的kwargs_change来修改参数名："target": "label"

由于逻辑比较公共，就无需一个单独的FunctionalKLDivMatcher了，上面的Matcher可以起一个公共的名字：ProcessReduceMatcher，其他API如果也是相同逻辑就复用。

如果逻辑比较特殊，比如这里的paddle.exp(target)，才考虑建立一个非公有的Matcher

File: tests/test_nn_functional_kl_div.py [hunk 2]
```diff
@@ -136,8 +136,7 @@ def test_case_9():
     obj.run(pytorch_code, ["result"])
 
 
-# result dismatch
-def _test_case_10():
+def test_case_10():
```
Comment (zhwesky2010) [inline tests/test_nn_functional_kl_div.py:139]: log_target=True 这里有没有测试

File: paconvert/api_matcher.py (hunk 1 above)
Comment (NKNaN) [reply paconvert/api_matcher.py]: 好的，已修改

File: tests/test_nn_functional_kl_div.py (hunk 2 above)
Comment (NKNaN) [reply tests/test_nn_functional_kl_div.py:139]: 已增加 log_target=True 的测试

**Review by zhwesky2010 — APPROVED** [review]
LGTM
